examples/2d/bloom_2d.rs

```diff
@@ -210,18 +180,33 @@ fn update_bloom_settings(
                 bloom.scale.x += dt * 2.0;
             }
             bloom.scale.x = bloom.scale.x.clamp(0.0, 16.0);
-
-            if keycode.just_pressed(KeyCode::KeyO) {
-                commands.entity(entity).insert(tonemapper.next());
-            }
         }
 
-        (entity, None) => {
-            text.0 = "Bloom: Off (Toggle: Space)".to_string();
+        (entity, _, None) => {
+            text.0 = "Bloom: Off (Toggle: Space)\n".to_string();
 
             if keycode.just_pressed(KeyCode::Space) {
                 commands.entity(entity).insert(Bloom::default());
             }
         }
     }
+
+    text.push_str(&format!("(O) Tonemapper: {:?}\n", bloom.1));
+    if keycode.just_pressed(KeyCode::KeyO) {
+        commands.entity(bloom.0).insert(next_tonemap(bloom.1));
+    }
+}
+
+/// Modifies resources tonemapping algorithm for the next one and returns it
+fn next_tonemap(tonemap: &Tonemapping) -> Tonemapping {
```

Comment:
Just a nit: we've got "Tonemapper" "tonemapping" and "tonemap" all referring to this value. Would love to standardize on "tonemapping."